lib/push.js

```diff
@@ -171,6 +171,40 @@ module.exports = function (log, db, config) {
   }
 
   /**
+   * iOS clients don't yet support all commands types, and due to
+   * platform limitations they have to show some bad fallback UX
+   * if they receive an unsupported message type.  Filter out
+   * devices that we know won't respond well to the given command.
+   *
+   * @param {String} command
+   * The command from the push message payload
+   * @param {Device[]} devices
+   * The list of devices to which to send the push.
+   */
+  function filterSupportedDevices(command, devices) {
+    var minVersion
```

Comment:
nit: `miniOsVersion` would be a little more clear.

Reply:
Please not `miniOsVersion`! That reads like "mini-OS version".

Reply:
Heh, yea didn't notice that until you pointed it out. My line of thinking was since this function could potentially filter any device from any OS it might be good to be specific in variable name here.

Maybe `minimumiOsVersion`?

Reply:
If the `ios` part has to be included I'd ignore Apple's upper/lowercase whimsy entirely. And for the prefix I think `required` > `minimum` > `min`. So `requiredIosVersion` is my vote, if this is a democracy.